jujugui/static/gui/src/app/components/shared/basic-table/basic-table.js

```diff
@@ -84,7 +86,7 @@ BasicTable.propTypes = {
   headers: PropTypes.arrayOf(PropTypes.shape({
     content: PropTypes.node,
     // The number of columns (between 1 and 12).
-    columnSize: PropTypes.number.isRequired,
```

Comment:
These are no longer required but I also don't see them being used...

Reply:
It's not - I've removed.